hknweb/reviewsessions/templates/reviewsessions/show_details.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
```

Comment:
This has a lot of common code with [events show details](https://github.com/compserv/hknweb/blob/master/hknweb/events/templates/events/show_details.html). Let's merge them into one template.

Reply:
[Jira ticket](https://hkn-mu.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=COMPSERV&modal=detail&selectedIssue=COMPSERV-40) created.